SvelteKit website crashing when deployed to Vercel

My website (https://github.com/electricservers/electricservers.com.ar) deployed in https://electricservers.com.ar, which [uses Gamedig to query a list of servers](https://github.com/electricservers/electricservers.com.ar/blob/main/src/routes/%2Blayout.server.ts) to show in the frontend, throws a 500 error when deployed (successfully) to Vercel, but works perfectly normal when running it locally, through dev AND building/preview.

**Where can I begin trying to fix this issue? Is this even the right place to ask?**

Vercel logs point to gamedig:

```
2022-12-28T18:16:33.972Z	26eed3a2-844b-476c-ad87-cc122de3d410	ERROR	Error: Failed all 1 attempts
    at QueryRunner.run (/var/task/node_modules/.pnpm/gamedig@4.0.5/node_modules/gamedig/lib/QueryRunner.js:95:21)
    at async Gamedig.query (/var/task/node_modules/.pnpm/gamedig@4.0.5/node_modules/gamedig/lib/index.js:11:16)
    at async Gamedig.query (/var/task/node_modules/.pnpm/gamedig@4.0.5/node_modules/gamedig/lib/index.js:19:16)
    at async load (file:///var/task/.svelte-kit/output/server/entries/pages/_layout.server.ts.js:27:5)
    at async load_server_data (file:///var/task/.svelte-kit/output/server/index.js:946:18)
    at async file:///var/task/.svelte-kit/output/server/index.js:1935:18
Attempt #1 - Port=27034 Retry=0:
Error: Cannot find module './lib/freeze'
Require stack:
- /var/task/node_modules/.pnpm/compressjs@1.0.3/node_modules/compressjs/main.js
- /var/task/node_modules/.pnpm/gamedig@4.0.5/node_modules/gamedig/protocols/valve.js
- /var/task/node_modules/.pnpm/gamedig@4.0.5/node_modules/gamedig/lib/ProtocolResolver.js
- /var/task/node_modules/.pnpm/gamedig@4.0.5/node_modules/gamedig/lib/QueryRunner.js
- /var/task/node_modules/.pnpm/gamedig@4.0.5/node_modules/gamedig/lib/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at req (/var/task/node_modules/.pnpm/amdefine@1.0.1/node_modules/amdefine/amdefine.js:144:31)
    at stringRequire (/var/task/node_modules/.pnpm/amdefine@1.0.1/node_modules/amdefine/amdefine.js:220:28)
    at amdRequire (/var/task/node_modules/.pnpm/amdefine@1.0.1/node_modules/amdefine/amdefine.js:113:24)
    at /var/task/node_modules/.pnpm/amdefine@1.0.1/node_modules/amdefine/amdefine.js:177:24
    at Array.map (<anonymous>)
    at runFactory (/var/task/node_modules/.pnpm/amdefine@1.0.1/node_modules/amdefine/amdefine.js:176:25)
    at define (/var/task/node_modules/.pnpm/amdefine@1.0.1/node_modules/amdefine/amdefine.js:277:13)
```

This definitely isn't a networking issue. I suspect there's something wrong with the way pnpm is installing the compressjs package. `Error: Cannot find module './lib/freeze'` makes it sound like the lib/freeze.js file is missing from compressjs, while that's definitely not the case: https://github.com/cscott/compressjs/blob/master/lib/freeze.js

I made some changes to the project by deleting all pnpm references, installing all packages through npm, and overriding the install command in Vercel to `npm install`
![image](https://user-images.githubusercontent.com/20617661/210014974-16d823ce-e20b-45e9-ba12-e062e7ddca4f.png)
Unfortunately the issue still persists:
```ts
[GET] /api/servers
18:58:02:66
2022-12-29T21:58:02.738Z	144bc8c4-0cb1-4772-aee4-00225bf16e0b	ERROR	Error: Failed all 1 attempts
    at QueryRunner.run (/var/task/node_modules/gamedig/lib/QueryRunner.js:95:21)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Gamedig.query (/var/task/node_modules/gamedig/lib/index.js:11:16)
    at async Gamedig.query (/var/task/node_modules/gamedig/lib/index.js:19:16)
    at async GET (file:///var/task/.svelte-kit/output/server/entries/endpoints/api/servers/_server.ts.js:4:16)
    at async render_endpoint (file:///var/task/.svelte-kit/output/server/index.js:280:22)
    at async resolve (file:///var/task/.svelte-kit/output/server/index.js:2248:22)
    at async respond (file:///var/task/.svelte-kit/output/server/index.js:2152:22)
    at async Server.default (file:///var/task/.svelte-kit/vercel-tmp/index.js:31:3)
Attempt #1 - Port=27034 Retry=0:
Error: Cannot find module './lib/freeze'
Require stack:
- /var/task/node_modules/compressjs/main.js
- /var/task/node_modules/gamedig/protocols/valve.js
- /var/task/node_modules/gamedig/lib/ProtocolResolver.js
- /var/task/node_modules/gamedig/lib/QueryRunner.js
- /var/task/node_modules/gamedig/lib/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at req (/var/task/node_modules/amdefine/amdefine.js:144:31)
    at stringRequire (/var/task/node_modules/amdefine/amdefine.js:220:28)
    at amdRequire (/var/task/node_modules/amdefine/amdefine.js:113:24)
    at /var/task/node_modules/amdefine/amdefine.js:177:24
    at Array.map (<anonymous>)
    at runFactory (/var/task/node_modules/amdefine/amdefine.js:176:25)
    at define (/var/task/node_modules/amdefine/amdefine.js:277:13)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,6 @@
 
 ### To Be Released...
+* Replace `compressjs` dependency by `seek-bzip` to solve some possible import issues.
 * Sons Of The Forest - Added support
 * Red Dead Redemption 2 - RedM (2018) - Added support
 * Creativerse (2017) - Added support

--- a/package.json
+++ b/package.json
@@ -38,13 +38,13 @@
     },
     "dependencies": {
         "cheerio": "^1.0.0-rc.10",
-        "compressjs": "^1.0.2",
         "gbxremote": "^0.2.1",
         "got": "^12.1.0",
         "iconv-lite": "^0.6.3",
         "long": "^5.2.0",
         "minimist": "^1.2.6",
         "punycode": "^2.1.1",
+        "seek-bzip": "^2.0.0",
         "varint": "^6.0.0"
     },
     "bin": {

--- a/protocols/valve.js
+++ b/protocols/valve.js
@@ -1,4 +1,4 @@
-const Bzip2 = require('compressjs').Bzip2;
+const Bzip2 = require('seek-bzip');
 const Core = require('./core');
 const Results = require('../lib/Results');
 const Reader = require('../lib/reader');
@@ -592,7 +592,7 @@ class Valve extends Core {
                     if(bzip) {
                         this.debugLog("BZIP DETECTED - Extracing packet...");
                         try {
-                            assembled = Buffer.from(Bzip2.decompressFile(assembled));
+                            assembled = Bzip2.decode(assembled);
                         } catch(e) {
                             throw new Error('Invalid bzip packet');
                         }

